blmath/geometry/transform/test_coordinate_manager.py

```diff
@@ -102,7 +102,9 @@ def test_coordinate_manager_forward_on_mesh(self):
 
         cube_v = create_cube_verts([1., 0., 0.], 4.)
         cube_floor_point = np.array([3., 0., 2.]) # as lace.mesh.floor_point
-        cube = MagicMock(v=cube_v, other_thing=np.array([-9.]))
+        # spec the mock for avaiable attributes since magic mock will "always"
```

Comment:
Just for clarity, would you change this comment to something like "By default a magic mock will always have any attribute it's asked for; here we set the spec property so that it will not respond to having a copy method when the CoodinateManager looks for it."